cmd/dp-import-tracker/main.go

```diff
@@ -205,7 +225,10 @@ func main() {
 		ObservationsInsertedTopic: "import-observations-inserted",
 		Brokers:                   []string{"localhost:9092"},
 		ImportAPIAddr:             "http://localhost:21800",
+		ImportAPIAuthToken:        "FD0108EA-825D-411C-9B1D-41EF7727F465",
 		DatasetAPIAddr:            "http://localhost:22000",
+		DatasetAPIAuthToken:       "FD0108EA-825D-411C-9B1D-41EF7727F465",
```

Comment:
Import API and dataset API auth tokens the same?

Reply:
isn't that normal?

Reply:
It looks like they have been made the same - so all good 👍